config.py

```diff
@@ -82,6 +82,15 @@ class Config(object):
         'g-cloud-5': 'g-cloud-6',
     }
 
+    # Note that because our GTM auth token is rendered into the client environment (see
+    # _javascripts.html), there isn't any point in encrypting it, as we would with other
+    # similar tokens.
```

Comment:
The `gtmEnvironmentIDShouldBeOverriddenByLocalConfig` confused me, as this seemed to be a deliberately obfuscating thing.